Libs/MRML/Core/vtkMRMLCoreTestingMacros.h

```diff
@@ -763,7 +763,7 @@
         ", green = " << node1->IsDisplayableInView(green) <<            \
         ", yellow = " << node1->IsDisplayableInView(yellow) <<          \
         ", 3D = " << node1->IsDisplayableInView(threeD) <<              \
-        std::cout;                                                      \
+        std::endl;                                                      \
```

Comment:
Could you updated the commit message to include the error fixed by this commit. Copy the error ouput of visual studio is enough.